examples/js/exporters/GLTFExporter.js

```diff
@@ -1358,14 +1358,16 @@ THREE.GLTFExporter.prototype = {
 
 				if ( geometry.index !== null ) {
 
-					if ( cachedData.attributes.has( geometry.index ) ) {
+					var cacheKey = geometry.uuid + ':' + groups[ i ].start + ':' + groups[ i ].count;
+
+					if ( cachedData.attributes.has( cacheKey ) ) {
 
 						primitive.indices = cachedData.attributes.get( geometry.index );
```

Comment:
Already merged and I'm fixing in #16267, but just in case. Not `cachedData.attributes.get( geometry.index )` but `cachedData.attributes.get( cacheKey )`.

Reply:
Thanks for that. Already fixed with #16272.